packages/common/scss/common.scss

```diff
@@ -101,3 +101,48 @@ $theme-site-header-breakpoints: (
     margin-top: 1.5rem;
   }
 }
+
```

Comment:
What's the difference between a `blockquote` HTML element and a `.pullquote` classed element? Trying to remember...

Reply:
They're just styled differently, but it's easier to show visually than explain - screenshots in description of both :)